src/components/form/Form.js

```diff
@@ -0,0 +1,124 @@
+import { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import InputContainer from './Form.styeld';
+import useStore from './useStore/useStore';
+import Wrapper from './Wrapper.styled';
+
+export default function Form() {
+	const addNewCard = useStore(state => state.addNewCard);
+	const {
+		register,
+		handleSubmit,
+		formState: { errors },
+	} = useForm();
+
+	const onSubmit = (data, event) => {
+		addNewCard(data);
+		event.target.reset();
+	};
+
+	const initialButtonText = 'Submit';
+	const [buttonText, setButtonText] = useState(initialButtonText);
+
+	const handleClick = () => {
+		setButtonText('Added');
+
+		setTimeout(() => {
+			setButtonText(initialButtonText);
+		}, 1000);
+	};
+
+	return (
+		<Wrapper>
+			<form onSubmit={handleSubmit(onSubmit)}>
+				<InputContainer>
+					<h4>Animal Profil</h4>
+
+					<div>
+						<input
+							id="name"
+							type="text"
+							aria-invalid={errors.name ? 'true' : 'false'}
+							{...register('name', {
+								required: 'Add animal name',
+								maxLength: 20,
+							})}
+							placeholder="Name"
+						/>
+					</div>
+					<div>
+						<input
+							id="health"
+							type="text"
+							{...register('health', {
+								required: 'Add health status ',
+								maxLength: 170,
+							})}
+							placeholder="Zustand"
+						/>
+					</div>
+					<div>
+						<input
+							id="behave"
+							type="text"
+							{...register('behave', {
+								required: 'Add health status',
```

Comment:
I think here's a copy/paste mistake: 
```suggestion
								required: 'Add behavior',
```

Reply:
is changed